docs/user/convert/parsing_and_validation.rst

```diff
@@ -201,16 +201,16 @@ by default. The reporters can be configured as follows:
 
     geomesa.converters.myconverter {
       options {
-        reporters = {
-          graphite = {
+        reporters = [
+          {
             type           = "graphite"
             url            = "localhost:9000"
             prefix         = "example"
             rate-units     = "SECONDS"
             duration-units = "MILLISECONDS"
             interval       = "10 seconds"
-          }
-          ganglia = {
+          },
```

Comment:
Do the docs line up with how to configure the different reporters?

Reply:
yeah, I believe so.

Reply:
the configs themselves didn't change, but the outer container changed from a named map to an array (since the names were not used/irrelevant)

Reply:
the named map will still work for backwards compatibility, but we don't document it.